test/integration/relay/connection.test.js

```diff
@@ -193,12 +201,12 @@ if (helper.sequelize.dialect.name === 'postgres') {
           orderBy: new GraphQLEnumType({
             name: 'Viewer' + this.Task.name + 'ConnectionOrder',
             values: {
-              ID: {value: [this.Task.primaryKeyAttribute, 'ASC']},
+              ID: {value: [this.Task.primaryKeyAttribute, 'ASC']}
             }
           }),
-          before: (options, args, root) => {
+          before: (options, args, context, {viewer}) => {
             options.where = options.where || {};
-            options.where.userId = root.viewer.get('id');
+            options.where.userId = viewer.get('id');
             return options;
```

Comment:
Any suggestions on how to do that? the test was failing when it was just using root, and root.viewer.get('id') saying viewer was not defined. Root would have been context in this case.